Purchases/SubscriberAttributes/SpecialSubscriberAttributes.swift

```diff
@@ -15,30 +15,33 @@
 import Foundation
 
 // swiftlint:disable identifier_name
-struct SpecialSubscriberAttributes {
-
-    static let email = "$email"
-    static let phoneNumber = "$phoneNumber"
-    static let displayName = "$displayName"
-    static let pushToken = "$apnsTokens"
-
-    static let idfa = "$idfa"
-    static let idfv = "$idfv"
-    static let gpsAdId = "$gpsAdId"
-
-    static let ip = "$ip"
-
-    static let adjustID = "$adjustId"
-    static let appsFlyerID = "$appsflyerId"
-    static let fBAnonID = "$fbAnonId"
-    static let mpParticleID = "$mparticleId"
-    static let oneSignalID = "$onesignalId"
-
-    static let mediaSource = "$mediaSource"
-    static let campaign = "$campaign"
-    static let adGroup = "$adGroup"
-    static let ad = "$ad"
-    static let keyword = "$keyword"
-    static let creative = "$creative"
+enum SpecialSubscriberAttribute: String, RawRepresentable {
+
+    var key: String { rawValue }
```

Comment:
this isn't needed, we can just use `rawValue`, but I found it more readable to do like `adjustID.key` than `adjustID.rawValue`. Happy to change if others don't like it, though

Reply:
I always found it awkward to use `rawValue` instead of something nice like `key` I appreciate this 😄